src/core/src/bootstrap/Constants.py

```diff
@@ -183,6 +183,7 @@ class StatusTruncationConfig(EnumBackport):
         MIN_ASSESSMENT_PATCHES_TO_RETAIN = 5
         TRUNCATION_WARNING_MESSAGE = "Package lists were truncated to limit reporting data volume. In-VM logs contain complete lists."
         TURN_ON_TRUNCATION = True
+        DELAY_TRUNCATION_BY_SEC = 60
```

Comment:
MIN_TRUNCATION_INTERVAL_IN_SEC

why - the variable as it is named now sounds like a one time delay. And it's not a constant delay, it's the minimum interval of time that must pass before truncation can happen normally after the last time (exceptions around terminal states exist of course)

Reply:
done